mix.lock

```diff
@@ -1,43 +1,160 @@
 %{
```

Comment:
this file should not be formatted, this is why it is not included in `.formatter.exs`